LF Spec: fix ordering of contract IDs. (#8389)
digital-asset/daml: daml-lf/spec/daml-lf-1.rst

enclose symbols such as ``.`` or ``→`` in quotes for the sake of
brevity and readability.

Identifiers
~~~~~~~~~~~

> In this section, we define a bunch of literals that can be handled by
> DAML-LF programs.

Reviewer: ```suggestion
In this section, we define the sorts of strings and identifiers that appear in 
DAML-LF programs.
```

    —————————————————————————————————————————————————————————————————————— EvLessEqTimestamp
      𝕆('LESS_EQ' @σ LitTimestamp₁ LitTimestamp₂) =

>           Ok (LitTimestamp₁ ≤ₗ≤ LitTimestamp₂)

Reviewer: ```suggestion
          Ok (LitTimestamp₁ ≤ₗ LitTimestamp₂)
```

    —————————————————————————————————————————————————————————————————————— EvLessEqNumeric
      𝕆('LESS_EQ' @σ LitNumeric₁ LitNumeric₂) =
          Ok (LitNumeric₁ ≤ₗ LitNumeric₂)

> 

Author: ```suggestion
```
